pkg/store/witness/witness.go

```diff
@@ -70,6 +70,8 @@ func (s *Store) Put(anchorID string, witnesses []*proof.Witness) error {
 
 	anchorIDEncoded := base64.RawURLEncoding.EncodeToString([]byte(anchorID))
 
+	putOptions := &storage.PutOptions{IsNewKey: true}
```

Comment:
I don't think we need this here. There will usually only be 2 or 3 items in the batch.

Reply:
It'll still be faster and more efficient to use this flag whenever possible. Is there some reason not to?

Reply:
In case a witness should happen to be added twice (for some reason) it will fail. So I don't think it's worth it.

Reply:
Actually, it's fine. I didn't notice that a UUID was used as the key.